roles/KubevirtNodeLabeller/defaults/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
 # defaults file for KubevirtNodeLabeller
 wait: true
-use_kvm: true
+useKVM: true
+use_kvm: false
```

Comment:
Honestly, I do not like the duplication.

Reply:
we have another option:
bury this into the task file, which will set `useKVM=true` also if the `use_kvm` var is true. This will allow us to leave the deployment and the default files unchanged.

Reply:
We need to check if HCO stable version X can or should consume our operator version X-1. If so, we will need the backward compat. Otherwise, we can happily change only our master and call it done without the ugly backward compat.